### PR TITLE
release-notes.txt: add 2022.01 release notes

### PR DESCRIPTION
**Pull request by emmanuelsearch**
### Contribution description

These are the release notes for the 2022.01 release. Please give it a look!

### Testing procedure

Make sure that your work in the last 3 months is present in the list ;)

### Issues/PRs references

See notes ;)

### CODE REVIEW

**Comment by aabadie** [comment]
> That's a really long list...

Note that the whole release notes are generated automatically using https://github.com/RIOT-OS/RIOT-release-manager/pull/20. This tool works with the GitHub labels that are set on PRs/Issues. For changes. all PRs merged that belong to the current milestone are listed.

The known issues are listed when they are labelled as `Type: bug`.
For PRs, it's a little more complex and probably impossible to have perfect and automatic grouping of PRs, knowing that labels are set automatically by the labeler GH action.
If we wanted a "Ztimer migration" group, we would need a corresponding label, to adat the labeler rules and also to adapt the script mentioned in the PR above.
Of course, it's still possible to generate the notes once and manually adapt afterwards.

File: release-notes.txt [hunk 1]
```diff
@@ -1,3 +1,791 @@
...
+
+Deprecations
+============
+
+Deprecations (2)
+----------------
+
+* cpu/mips: deprecate mips cpu (#17304)
+* sys/include/xtimer.h: deprecate nanosleep (#17296)
+
```
Comment (kfessel) [inline release-notes.txt:401]:
```suggestion
* sys/include/evtimer.h: deprecate evtimer_now_min (#17357)

```

File: release-notes.txt [hunk 2]
```diff
@@ -1,3 +1,791 @@
...
+The 2022.01 release includes:
+
+## List of notable changes here ##
+
+436 pull requests, composed of 1159 commits, have been merged since the
+last release, and 11 issues have been solved. 43 people contributed with
+code in 88 days. 2475 files have been touched with 44925 (+) insertions and
+14687 deletions (-).
+
+
```
Comment (fjmolinas) [inline release-notes.txt:30]: I would mention here: all the work regarding `rust`, that almost all `boards` where migrated to kconfig. The fact that with `ztimer64` all code can be migrated to `ztimer` from `xtimer`.

File: release-notes.txt [hunk 3]
```diff
@@ -1,3 +1,791 @@
...
+* sys/random: default to musl LCG instead of TinyMT (#17188)
+* sys/suit: adjust dependencies for CoAP transport (#17182)
+* sys/test_utils/result_output/json: optional space after symbol (#17034)
+* sys/turo: Allow default selection of json (#17039)
+* sys/ztimer/xtimer2ztimer.cocci: extend and fix api replacements (#17234)
+* sys/ztimer64: default select ztimer64_init (#17414)
+* sys/ztimer64: make _del_entry_from_list() safe for uninit ztimer64_t (#17368)
+* sys/ztimer: remove double ztimer indirection (#17272)
+* sys/ztimer: ztimer_remove report success (#17408)
+* usbdev/USBUS: deprecate transfer failure reporting report (#17046)
```
Comment (fjmolinas) [inline release-notes.txt:94]: I would move this to deprecations...

File: release-notes.txt [hunk 4]
```diff
@@ -1,3 +1,791 @@
...
++ sys/stdio: add optional function stdio_available (#17446)
++ sys/test_utils/result_output: add sort to multiple output check (#17269)
++ sys/tsrb: add peek functions (#17266)
++ sys/tsrb: add tsrb_clear (#17337)
++ sys/ztimer: add 'ztimer_no_periph_rtt' (#17284)
++ sys/ztimer: add some best practices (#17370)
++ sys/ztimer: ztimer_now() add warning regarding comparing now() values (#17404)
++ usb/hid: Add HID report descriptor defines (#17242)
+* drivers/periph_common: build periph module selectively (#17418)
+* event/timeout: remove forced ZTIMER_USEC dependency [alternative] (#16958)
```
Comment (fjmolinas) [inline release-notes.txt:69]:
```suggestion
* event/timeout: remove forced ZTIMER_USEC dependency (#16958)
```

File: release-notes.txt [hunk 5]
```diff
@@ -1,3 +1,791 @@
...
++ pkg: add support for Elk Tiny Javascript engine (#16887)
++ pkg: add support for FreeRTOS coreJSON library (#16997)
+* hacl: Switch git address to RIOT-OS-pkgmirror (#16941)
+* make: support package mirrors (#16927)
+* pkg/emlearn: bump to 0.12 (#16937)
+* pkg/jerryscript: bump to 2.4.0 (#16938)
+* pkg/mynewt-core/patches: silence cast-align (#17397)
+* pkg/qr-code-generator: bump to 1.7.0 (#16939)
+* pkg/uwb-core: fix wrong header include (#17345)
+* pkg/uwb-core: multiple cleanups (#17264)
```
Comment (fjmolinas) [inline release-notes.txt:147]:
```suggestion
* pkg/uwb-core: fix wrong header include (#17345)
* pkg/uwb-core: enable frame filtering, allow usage o common event thread (#17264)
```

**Review by fjmolinas — COMMENTED** [review]
Some first comments will take a better look later.

> Note that the whole release notes are generated automatically using RIOT-OS/RIOT-release-manager#20. This tool works with the GitHub labels that are set on PRs/Issues. For changes. all PRs merged that belong to the current milestone are listed.

Awesome to this being used, will help future release notes a bunch! Its a great spet forward formalizing the release notes structure a bit IMO. In that direction in past releases API changes have been inlined in the release notes, is this something we want to always include?

>  Should we distinguish between relevant and irrelevant entries? 

I would be rather in favor of some pruning of items on the list if this is hard to agree on then keeping everything might be the simplest. I do think some PR titles are not descriptive enough, so probably good to take a look at all items (fix typos as well).

My TOC is also telling me that it would like all titles to not have use caps at the begining (or all yes)

File: release-notes.txt (hunk 2 above)
Comment (emmanuelsearch) [reply release-notes.txt:30]: Yup. Draft summary in the pipe. Stay tuned (and thanks for the suggestions @fjmolinas )

**Comment by emmanuelsearch** [comment]
Fixed some static-test errors (codespell) and squashed immediately

**Comment by emmanuelsearch** [comment]
Aside the debate concerning exhaustiveness, any comments or suggestions on the summary/highlights in the draft release notes? @chrysn @leandrolanzieri @fjmolinas @all-actually ;)

File: release-notes.txt (hunk 1 above)
Comment (kfessel) [reply release-notes.txt:413]: @emmanuelsearch this

File: release-notes.txt [hunk 6]
```diff
@@ -1,3 +1,803 @@
...
+
++ core/include/kernel_defines.h: add index_of() macro (#17142)
++ core: Add IS_CT_CONSTANT() (#17273)
++ core: add WITHOUT_PEDANTIC() and DECLARE_CONSTANT() (#17176)
+* core: Warn about using irq_enable (#17433)
+
+System Libraries
+----------------
+
++ pkg/fff: Add fake functions framework package (#17076)
```
Comment (aabadie) [inline release-notes.txt:78]: Should rather be listed in the pkg section

File: release-notes.txt [hunk 7]
```diff
@@ -1,3 +1,803 @@
...
++ sys/sched_rr: Add a round robin scheduler module (#16126)
++ sys/stdio: add optional function stdio_available (#17446)
++ sys/test_utils/result_output: add sort to multiple output check (#17269)
++ sys/tsrb: add peek functions (#17266)
++ sys/tsrb: add tsrb_clear (#17337)
++ sys/ztimer: add 'ztimer_no_periph_rtt' (#17284)
++ sys/ztimer: add some best practices (#17370)
++ sys/ztimer: ztimer_now() add warning regarding comparing now() values (#17404)
++ usb/hid: Add HID report descriptor defines (#17242)
+* drivers/periph_common: build periph module selectively (#17418)
```
Comment (aabadie) [inline release-notes.txt:93]: Should go in the drivers section

File: release-notes.txt [hunk 8]
```diff
@@ -1,3 +1,803 @@
...
+* event/timeout: remove forced ZTIMER_USEC dependency (#16958)
+* examples/gnrc_networking: cleanup and static function declarations in
```
Comment (aabadie) [inline release-notes.txt:95]: Should go in the examples section

File: release-notes.txt [hunk 9]
```diff
@@ -1,3 +1,803 @@
...
+  udp.c (#17384)
+* pkg/arduino_sdi_12: support the remote-revb board (#17119)
+* pkg/fatfs: bump version to r0.14b (#16412)
```
Comment (aabadie) [inline release-notes.txt:98]: Both should go to pkg section

File: release-notes.txt [hunk 10]
```diff
@@ -1,3 +1,803 @@
...
+* usbus: check received setup request data amount (#17203)
+
+Networking
+----------
+
++ gnrc_pktbuf_cmd: add od dependency with gnrc_pktbuf_static (#17228)
++ sys/net/dhcpv6: Add stateless DHCPv6 (#16731)
++ sys/net: Add ipv4/ipv6 pseudomodules (#16966)
++ sys/shell/sc_nimble_netif: add ping subcommand (#16539)
+* doc: fix missing or non matching arguments in Doxygen documentation (#17461)
```
Comment (aabadie) [inline release-notes.txt:132]: Move to doc section

File: release-notes.txt [hunk 11]
```diff
@@ -1,3 +1,803 @@
...
+  configure the build (#16948)
+* pkg/openthread: migrate to ztimer (#17120)
+* pkg/paho-mqtt: migrate to ztimer (#17143)
+* pkg/semtech-loramac: fix get/set dr command (#17117)
+* sys/evtimer,ztimer: do not depend on ztimer_now64 (#17357)
+* sys/net/gnrc: fix compilation with -Wcast-align (#17157)
+* sys/shell/gnrc_netif: Allow 'ifconfig help' (#17006)
+* test/ieee802154_hal: general cleanup and cosmetic fixes (#16913)
+* tests/lwip_sock: Take lock before calling etharp (#17145)
+* Update tiny-asn1 package (#16936)
```
Comment (aabadie) [inline release-notes.txt:155]: Should move to pkg section

File: release-notes.txt [hunk 12]
```diff
@@ -1,3 +1,803 @@
...
+* make: support package mirrors (#16927)
+* pkg/emlearn: bump to 0.12 (#16937)
+* pkg/jerryscript: bump to 2.4.0 (#16938)
+* pkg/mynewt-core/patches: silence cast-align (#17397)
+* pkg/qr-code-generator: bump to 1.7.0 (#16939)
+* pkg/uwb-core: fix wrong header include (#17345)
+* pkg/uwb-core: enable frame filtering, allow usage of common event thread (#17264)
+* pkg/wakaama: use ztimer (#17103)
+* test/pkg_u8g2: increase main stacksize for SDL (#16954)
+* tests/pkg_fatfs{_vfs}: drop whitelist (#17478)
```
Comment (aabadie) [inline release-notes.txt:174]: Should go to the Testing section

File: release-notes.txt [hunk 13]
```diff
@@ -1,3 +1,803 @@
...
++ boards/stm32f469i-disco: Add DAC (#17214)
++ boards/stm32g031-disco,examples,tests: Added STM32G031-DISCO board (#16959)
++ boards: add board support for Alientek Pandora (#17259)
++ boards: add support for stm32f746g-disco (#17198)
++ cpu/esp32: Add openocd programming support (#16911)
++ cpu/stm32: add driver for the LTDC peripheral (#17437)
++ cpu/stm32: add new usbdev FS support (#17281)
++ cpu/stm32: add support for U5 family (#17410)
++ drivers/ft5x06: add support for touch panel controller (#17448)
++ drivers/stmpe811: add spi mode (#17088)
```
Comment (aabadie) [inline release-notes.txt:193]: Should move to the Drivers section

File: release-notes.txt [hunk 14]
```diff
@@ -1,3 +1,803 @@
...
++ boards/nucleo-g431rb: added PWM configuration (#17030)
++ boards/p-l496g-cell02: add riotboot feature (#17463)
++ boards/stm32f469i-disco: Add DAC (#17214)
++ boards/stm32g031-disco,examples,tests: Added STM32G031-DISCO board (#16959)
++ boards: add board support for Alientek Pandora (#17259)
++ boards: add support for stm32f746g-disco (#17198)
++ cpu/esp32: Add openocd programming support (#16911)
++ cpu/stm32: add driver for the LTDC peripheral (#17437)
++ cpu/stm32: add new usbdev FS support (#17281)
++ cpu/stm32: add support for U5 family (#17410)
```
Comment (aabadie) [inline release-notes.txt:191]: Should move to the CPU section

File: release-notes.txt [hunk 15]
```diff
@@ -1,3 +1,803 @@
...
++ drivers/ft5x06: add support for touch panel controller (#17448)
++ drivers/stmpe811: add spi mode (#17088)
++ makefiles/cflags.inc.mk: Add -Wcast-align (#14955)
++ pkg/arduino_sdi_12: add SDI-12 for Arduino as package (#16587)
```
Comment (aabadie) [inline release-notes.txt:195]: Should move to Packages section

File: release-notes.txt [hunk 16]
```diff
@@ -1,3 +1,803 @@
...
++ boards/stm32g031-disco,examples,tests: Added STM32G031-DISCO board (#16959)
++ boards: add board support for Alientek Pandora (#17259)
++ boards: add support for stm32f746g-disco (#17198)
++ cpu/esp32: Add openocd programming support (#16911)
++ cpu/stm32: add driver for the LTDC peripheral (#17437)
++ cpu/stm32: add new usbdev FS support (#17281)
++ cpu/stm32: add support for U5 family (#17410)
++ drivers/ft5x06: add support for touch panel controller (#17448)
++ drivers/stmpe811: add spi mode (#17088)
++ makefiles/cflags.inc.mk: Add -Wcast-align (#14955)
```
Comment (aabadie) [inline release-notes.txt:194]: Should move to the Build system section

File: release-notes.txt [hunk 17]
```diff
@@ -1,3 +1,803 @@
...
++ pkg/arduino_sdi_12: add SDI-12 for Arduino as package (#16587)
++ sys/arduino: add Serial over stdio support (#17447)
++ sys: introduce sysclk function to retrieve core clock frequency (#17342)
```
Comment (aabadie) [inline release-notes.txt:197]: Should move to System section

File: release-notes.txt [hunk 18]
```diff
@@ -1,3 +1,803 @@
...
+* boards/lora-e5-dev: fix default DARWIN port (#17373)
+* boards/stm32f469i-disco: Create  board (#16898)
+* boards/{calliope-mini,microbit*}: factorize common microbit module,
+  use ztimer (#17124)
+* boards: cpu: replace GPIO_UNDEF by SPI_CS_UNDEF in SPI configuration (#17468)
+* boards: model nrf52 boards (#17299)
+* boards: unify user button defines (#17428)
+* cpu/esp: compile optional modules in CI (#17314)
+* cpu/stm32/wl: initial periph_adc implementation (#17183)
+* cpu/stm32: enable USB OTG FS on high end L4 (#17302)
```
Comment (aabadie) [inline release-notes.txt:210]: Should move to CPU

File: release-notes.txt [hunk 19]
```diff
@@ -1,3 +1,803 @@
...
+* socket_zep: port to radio HAL (#16932)
```
Comment (aabadie) [inline release-notes.txt:211]: Maybe move to Networking ?

File: release-notes.txt [hunk 20]
```diff
@@ -1,3 +1,803 @@
...
+
+CPU
+---
+
++ Add some Rust building infrastructure and example (#16274)
++ cpu/esp32: add clear bus during init to the I2C software
+  implementation (#17346)
++ cpu/nrf9160: add periph_flashpage support (#17225)
++ cpu/stm32: Add hardening changes to stm32 (#16979)
++ drivers/dose: make use of UART collision detection feature (#16681)
```
Comment (aabadie) [inline release-notes.txt:222]: Move to Drivers

File: release-notes.txt [hunk 21]
```diff
@@ -1,3 +1,803 @@
...
+* cpu/stm32: split periph_cpu.h in sub headers (#17441)
+* cpu: do not auto-select ztimer_periph_rtt for sam0, fe310 (#17395)
+* drivers/periph: use uint_fast8_t as default type (#17191)
+* esp8266: Download Espressif RTOS SDK as a new RIOT PKG (#16425)
+* Fix some cppcheck errors (#17271)
+* ieee802154/hal: adapt to latest changes of #13943 (#16946)
+* periph/flashpage: extend API (#16972)
+* periph/flashpage: fix wrong doxygen group (#17202)
+* stm32/usbdev: Use ztimer instead of xtimer (#17047)
+* sys/ztimer64: initial PR (#16928)
```
Comment (aabadie) [inline release-notes.txt:260]: Move to Sys

File: release-notes.txt [hunk 22]
```diff
@@ -1,3 +1,803 @@
...
+* treewide: Address uncrustify suggestions (#16737)
+* usb(dev|bus): Fix static-check issues (#17135)
```
Comment (aabadie) [inline release-notes.txt:262]: Maybe move to Testing (sounds more like CI fixes)

File: release-notes.txt [hunk 23]
```diff
@@ -1,3 +1,803 @@
...
+* drivers/sdcard_spi: convert to ztimer_usec (#17361)
+* drivers/stmpe811: improve interrupt callback management (#17487)
+* drivers/xbee: migrate to ztimer_msec (#17114)
+* drivers: migrate mpu9x50 and si70xx to ztimer (#17358)
+* drivers: migrate some drivers to ztimer (#17315)
+* drivers: remove some useless xtimer includes (#17303)
+* drivers: several cleanups related to xtimer + some ztimer migration (#17375)
+* makefiles: remove PCF857x pseudomodule definitions (#17343)
+* pkg/lvgl: use ztimer instead of xtimer (#17031)
+* pkg: migrate some packages to ztimer (#17309)
```
Comment (aabadie) [inline release-notes.txt:291]: Move to packages

**Review by aabadie — COMMENTED** [review]
Went through the list of changes and found some that should me moved to a section that fits more.

File: release-notes.txt (hunk 6 above)
Comment (aabadie) [reply release-notes.txt:78]: Or even "Testing"

File: release-notes.txt [hunk 24]
```diff
@@ -1,3 +1,805 @@
...
+
+RIOT now offers convenient integration of Rust application programming, with
+wrappers interfacing to RIOT modules such as timers, network stack;
+
+
+Improved timers
+---------------
+
+Many modules have migrated from using xtimer to our more power-efficient
+alternative ztimer. Also, ztimer64 was introduced, which brings paves the way
```
Comment (aabadie) [inline release-notes.txt:36]:
```suggestion
alternative ztimer. Also, ztimer64 was introduced, which paves the way
```

File: release-notes.txt [hunk 25]
```diff
@@ -1,3 +1,805 @@
...
+
++ Add some Rust building infrastructure and example (#16274)
++ cpu/esp32: add clear bus during init to the I2C software
+  implementation (#17346)
++ cpu/nrf9160: add periph_flashpage support (#17225)
++ cpu/stm32: Add hardening changes to stm32 (#16979)
++ nimble_scanner: rework to enable scanning BLE5 PHYs and extended
+  advertisements (#16843)
++ nrf5x_common: Add qdec peripheral implementation (#17201)
++ sys/new_delete: add malloc/free based new/delete implementation (#17464)
```
Comment (aabadie) [inline release-notes.txt:213]: Should be moved to sys (missed that one yesterday)

File: release-notes.txt [hunk 26]
```diff
@@ -1,3 +1,805 @@
...
++ cpu/esp32: Add openocd programming support (#16911)
++ cpu/stm32: add driver for the LTDC peripheral (#17437)
++ cpu/stm32: add new usbdev FS support (#17281)
++ cpu/stm32: add support for U5 family (#17410)
+* boards/common/esp8266: Set RTS and DTR to 0 on pyterm. (#16305)
+* cc2538_rf: remove RX Busy detection in request_set_trx_state (#16821)
+* core: make SCHED_TEST_STACK boolean and default to 1 with DEVELHELP (#17132)
```
Comment (aabadie) [inline release-notes.txt:220]: Should be moved to core

File: release-notes.txt [hunk 27]
```diff
@@ -1,3 +1,805 @@
...
+* cpu/stm32/include/cpu_conf_stm32_common.h: fix typo in macro (#17187)
+* cpu/stm32: bump cmsis repositories to latest releases + extend CPU
+  models kconfig (#17293)
+* cpu/stm32: duplicated ADC devices definitions (#16968)
+* cpu/stm32: migrate adc and eth periphs to ztimer (#17320)
+* cpu/stm32: split periph_cpu.h in sub headers (#17441)
+* cpu: do not auto-select ztimer_periph_rtt for sam0, fe310 (#17395)
+* drivers/periph: use uint_fast8_t as default type (#17191)
+* esp8266: Download Espressif RTOS SDK as a new RIOT PKG (#16425)
+* Fix some cppcheck errors (#17271)
```
Comment (aabadie) [inline release-notes.txt:246]: Should be moved to Testing

File: release-notes.txt [hunk 28]
```diff
@@ -1,3 +1,805 @@
...
+* ieee802154/hal: adapt to latest changes of #13943 (#16946)
```
Comment (aabadie) [inline release-notes.txt:247]: Should be moved to Networking

File: release-notes.txt [hunk 29]
```diff
@@ -1,3 +1,805 @@
...
+* cpu/kinetis: model kconfig (#17235)
+* cpu/lpc23xx: model kconfig (#17279)
+* cpu/nrf51: model kconfig (#17274)
+* cpu/qn908x: model kconfig (#17277)
+* cpu/{cc26xx, cc13xx}: model kconfig (#17236)
+* drivers/sx126x/Kconfig: rework model selection (#17289)
+* sys/benchmark/kconfig: cleanup ztimer dependency (#17317)
+* sys/log: model Kconfig (#17286)
+* sys/vfs: model Kconfig (#17287)
+
```
Comment (aabadie) [inline release-notes.txt:363]:
```suggestion
```

**Comment by emmanuelsearch** [comment]
@aabadie thanks for the details! Went through and performed the changes. Also, I point to the PRs merged with the milestone in the notes, to increase convenience: https://github.com/RIOT-OS/RIOT/milestone/37?closed=1

File: release-notes.txt [hunk 30]
```diff
@@ -1,3 +1,805 @@
...
+Bug fixes (66)
+==============
+
+* cpu/native: add `-no-pie` to LINKFLAGS (#16186)
+* boards/adafruit-clue: use internal RC oscillator (#17069)
+* boards/feather-m0-*: fix Arduino feature in Kconfig (#17444)
+* boards/nrf9160dk: fix LED macros (#17328)
+* boards/nucleo-g07xrb: fix or exclude some doxygen warnings (#17079)
+* boards/nucleo-g07xrb: fix or exclude some doxygen warnings [backport
+  2021.10] (#17084)
```
Comment (aabadie) [inline release-notes.txt:438]: There's a duplicate here. I would drop the 2021.10 backport.
```suggestion
* boards/nucleo-g07xrb: fix or exclude some doxygen warnings (#17079)
```

**Comment by emmanuelsearch** [comment]
oops, didn't see your latest  suggestion @aabadie, will do.

**Comment by emmanuelsearch** [comment]
anyone up for an ACK at this point?

File: release-notes.txt [hunk 31]
```diff
@@ -1,3 +1,802 @@
...
+
+The 2022.01 release brings support for new packages, new boards and drivers,
+as well as several pkg upgrades and bug fixes. Among others, this release
+improves the RIOT codebase with:
+
+Rust support
+------------
+
+RIOT now offers convenient integration of Rust application programming, with
+wrappers interfacing to RIOT modules such as timers, network stack;
```
Comment (fjmolinas) [inline release-notes.txt:29]:
```suggestion
wrappers interfacing to RIOT modules such as timers, network stack.
```

File: release-notes.txt [hunk 32]
```diff
@@ -1,3 +1,802 @@
...
+
+
+Improved timers
+---------------
+
+Many modules have migrated from using xtimer to our more power-efficient
+alternative ztimer. Also, ztimer64 was introduced, which paves the way
+for deprecating xtimer soon;
```
Comment (fjmolinas) [inline release-notes.txt:37]:
```suggestion
for deprecating xtimer soon.
```

Comment (fjmolinas) [inline release-notes.txt:37]:
```suggestion
alternative ztimer. Also, ztimer64 was introduced, which allows for
a drop-in replacement for all of xtimer features.
```

**Review by fjmolinas — APPROVED** [review]
Just a couple of nitpicks, otherwise LGTM.

File: release-notes.txt [hunk 33]
```diff
@@ -1,3 +1,802 @@
...
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2022.01 release brings support for new packages, new boards and drivers,
+as well as several pkg upgrades and bug fixes. Among others, this release
```
Comment (aabadie) [inline release-notes.txt:22]: One last nit:

```suggestion
as well as several package upgrades and bug fixes. Among others, this release
```

File: release-notes.txt (hunk 30 above)
Comment (aabadie) [reply release-notes.txt:435]: Still not addressed

File: release-notes.txt [hunk 34]
```diff
@@ -1,3 +1,802 @@
...
+* drivers/sdcard_spi: convert to ztimer_usec (#17361)
+* drivers/stmpe811: improve interrupt callback management (#17487)
+* drivers/xbee: migrate to ztimer_msec (#17114)
+* drivers: migrate mpu9x50 and si70xx to ztimer (#17358)
+* drivers: migrate some drivers to ztimer (#17315)
+* drivers: remove some useless xtimer includes (#17303)
+* drivers: several cleanups related to xtimer + some ztimer migration (#17375)
+* makefiles: remove PCF857x pseudomodule definitions (#17343)
+* drivers/periph_common: build periph module selectively (#17418)
+
```
Comment (aabadie) [inline release-notes.txt:285]:
```suggestion
```

File: release-notes.txt [hunk 35]
```diff
@@ -1,3 +1,802 @@
...
+* cpu: do not auto-select ztimer_periph_rtt for sam0, fe310 (#17395)
+* drivers/periph: use uint_fast8_t as default type (#17191)
+* esp8266: Download Espressif RTOS SDK as a new RIOT PKG (#16425)
+* periph/flashpage: extend API (#16972)
+* periph/flashpage: fix wrong doxygen group (#17202)
+* stm32/usbdev: Use ztimer instead of xtimer (#17047)
+* cpu/esp: compile optional modules in CI (#17314)
+* cpu/stm32/wl: initial periph_adc implementation (#17183)
+* cpu/stm32: enable USB OTG FS on high end L4 (#17302)
+
```
Comment (aabadie) [inline release-notes.txt:253]:
```suggestion
```

**Comment by bergzand** [comment]
And go!

File: release-notes.txt (hunk 30 above)
Comment (aabadie) [reply release-notes.txt:434]: Still not addressed (and a couple below as well)...

**Comment by emmanuelsearch** [comment]
Backport provided in #17585

**Pull request merged**